### ISSUE TITLE
setZone meaning ambiguity

### ISSUE THREAD

**Issue by hunterpankey** [moment/luxon#1012]
I just fought my way through an "as if it were" situation and needed to use fromISO() setZone=true, and expected the setZone parameter to work exactly the opposite of how it does.

**To Reproduce**
`let start: DateTime = DateTime.fromISO("2021-08-21T00:00:00Z", { setZone: false });`

**Actual vs Expected behavior**
I expected to get back a DateTime still in UTC, i.e., parse that datetime, but please don't set the time zone to my local one. But when using "setZone: false", that actually means, "please don't set the time zone to the one from the string". This is probably more trouble than it's worth, but it would be very helpful if the option were named "preserveZone" or "preserveSourceZone" (so that I should pass "true") or "convertToLocalZone" (so that I should pass "false") in order to make the intent and the outcome explicit.

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: 92.0.4515.159 (Official Build) (64-bit) (cohort: Stable)
- Luxon version: 2.0.1
- Your timezone: "America/New_York"

**Additional context**
Note: This is not a bug, just a thing that made it more difficult to get my situation handled properly.

**Comment by icambron** [comment]
Yeah, it's never been a great parameter name. We certainly want converting to the local zone to be the default (it is the right thing to do most of the time, since you know what zone your process is running in, but often not what strings you'll get). And I'm a big believer in the idea that boolean defaults should be `false`. But we could certainly name it better.

**Comment by uehlbran** [comment]
Would also be nice if it actually worked.

For example, if you have a variable with a string value of "America/Denver" and you pass that variable into the setZone method it does **** all but, if you pass in the string literal without storing it in a variable it all the sudden works.

**Comment by diesieben07** [comment]
@Abion47 There are arguments for both ways.
I can definitely see your argument for "don't mutate data by default", but you might also see it not as "mutating" but as "normalizing".

This is something I do want to revisit for version 4.0 though.
